packages/twenty-front/src/modules/object-record/hooks/useDetachMorphRelatedRecordFromRecord.ts

```diff
@@ -61,15 +61,8 @@ export const useDetachMorphRelatedRecordFromRecord = () => {
 
         if (isDefined(parentRecord)) {
           relatedObjectMetadataItems.forEach((relatedObjectMetadataItem) => {
-            const computedFieldName = computeMorphRelationFieldName({
```

Comment:
don't worry too much about this section of the code, it's ongoing work and we will likely re-work it